source/leuven/lecturer_s_procedure_to_request_student_accounts_ku_leuven_uhasselt.rst

```diff
@@ -3,30 +3,39 @@
 Lecturer's procedure to request student accounts (KU Leuven/UHasselt)
 =====================================================================
 
-In order to smoothly go through account creation for students process
-several actions from the lecturer are required.
+We support using HPC for educational purposes, and we fully assist the lecturers
+with all aspects of using HPC in the classroom, such as creating studing accounts,
+granting credits, resource reservation (if needed), and also live technical support
+during the trainings.
 
-#. Submit the request to
+In order to make this experience as convenient as possible for the students and the
+for the lecturer, it is requested that the lecturer (or any course responsible) would
+take the following actions:
+
+#. Submit a request to
    `hpcinfo@.kuleuven.be <mailto:hpcinfo@kuleuven.be?subject=Accounts%20requests%20for%20students%20attending%20the%20course>`_
-   providing a short description of the course and explanation why HPC
-   facilities are necessary for teaching the course. Please also add the
-   attachment with the list of students (including student number, provided by the toledo export)
    1-2 weeks before the beginning of the course.
-#. We advise to use :ref:`Open OnDemand <ood_t2_leuven>` service for the student to get access to the login nodes,
-   file browser and the job submission. For working with Open OnDemand only, the accounts are provisioned without students being involved in the procedure
-   and they login  to the cluster only using KU Leuven MFA.
-#. Since introductory credits are supposed to be used for private
-   projects (e.g. master thesis computations) we encourage to create
-   a specific project, starting with ``p_edu_``, which will be used for computations related to the course.
-   We will assign compute credits to this project. It also gives a lecturer an opportunity of tracing the use of
-   the cluster during the course. After provisioning the student accounts, we will also add these to the given project.
-   For more information about the procedure of creating the project please refer to the page
+   Please provide a short description of the course, and explain why the HPC
+   facilities are necessary for teaching the course. It is mandatory to provide a list
+   of students with their full names, KU Leuven/UHasselt email addresses, and their student number.
+   For KU Leuven students, you may convert the students' q-number to their r-/s-number using the export
+   tool on Toledo (Administrations -> Convert user-ids (q-numbers))
```

Comment:
Just jumping in to mention that once https://gitea.icts.kuleuven.be/foz/foz-vsc-accountpage-automation/pulls/7 is approved, the conversion of q-numbers is no longer necessary: the updated account creation script can deal with q-numbers.

Reply:
Indeed, this page will require another quick fix after that PR is merged and comes into production.